examples/clusterscoped/escape-to-host-si-sib-mixed.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: intent.security.nimbus.com/v1
+kind: SecurityIntent
+metadata:
+  name: escape-to-host
+spec:
+  intent:
+    id: escapeToHost
+    description: "A attacker can breach container boundaries and can gain access to the host machine"
+    action: Block
+    params:
+      exclude_resources: ["app:nginx"]
```

Comment:
This basically the list of labels which are to be excluded. This list should be specified in the intent binding, and not on the intent.